graphs/possible_bipartition.py

```diff
@@ -8,5 +8,25 @@ def possible_bipartition(dislikes):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+
+    if len(dislikes) == 0:
+        return False
+    first_item = list(dislikes.keys())[1]
+    stack = [first_item]
+    result = True
+    group_1 = []
+    group_2 = []
+    
+    for pet in dislikes:
+        group_1.append(dislikes[pet])
```

Comment:
`dislikes[pet]` is a list of nodes. I believe you may have intended to append each element of the `dislikes[pet]` individually to `group_1` instead